### PR TITLE
fixed ngcc error

### PR DESCRIPTION
**Pull request by sei-aschlackman**
- fixed up import/export statements to fix an error when compiling with ngcc
- added .prettierrc

### CODE REVIEW

**Comment by sei-aschlackman** [comment]
This seems to fix the ngcc error. I'm not 100% sure what was going on, but I noticed that the cmusei-crucible-common.d.ts in the dist folder had a lot of duplicate exports that should have been covered by public-api. These changes fix that, which must have been the issue.

Before:
image


After:
image

File: projects/@crucible-common/package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cmusei/crucible-common",
-  "version": "0.0.8",
+  "version": "0.0.9",
```
Comment (sei-ebram) [inline projects/@crucible-common/package.json:3]: this is not necessary - releasing the plugin updates this.  but it's fine, it'll just overwrite the same thing

Comment (sei-aschlackman) [reply projects/@crucible-common/package.json:3]: whoops! thanks for the info.

**Comment by sei-ebram** [comment]
Awesome find!

**Review by sei-ebram — APPROVED** [review]

**Review by sei-noconnor — APPROVED** [review]

**Pull request merged**